frontend/templates/troubleshooting/server.tsx

```diff
@@ -53,25 +54,33 @@ export const getServerSideProps: GetServerSideProps<
                      ifixitOrigin,
                   },
                   { forceMiss: hasDisableCacheGets(context) }
-               )
-         )
+               ).catch(() => null)
+            );
+         })
       );
       return {
          ...solution,
-         guides,
+         guides: guides.filter((guide): guide is Guide => Boolean(guide)),
          products: products.filter((product): product is ProductType =>
             Boolean(product)
          ),
       };
    }
 
-   const troubleshootingData = await client.get<TroubleshootingApiData>(
-      `Troubleshooting/${wikiname}`,
-      'troubleshooting',
-      {
-         credentials: 'include',
-      }
-   );
+   let troubleshootingData: TroubleshootingApiData;
+   try {
+      troubleshootingData = await client.get<TroubleshootingApiData>(
+         `Troubleshooting/${wikiname}`,
+         'troubleshooting',
+         {
+            credentials: 'include',
```

Comment:
This... shouldn't matter I don't think. This is server-side code, there's no cookies to send. I know your pull didn't add this, but I hadn't seen this code before. I'll likely open another issue to handle this.

Reply:
Issue created: https://github.com/iFixit/ifixit/issues/48080